resources/shuttle-aws-rds.mdx

```diff
@@ -31,6 +31,26 @@ async fn tide(#[shuttle_aws_rds::Postgres(
     )] pool: PgPool) -> ShuttleTide<MyState> { ... }
 ```
 
+The URI should be formatted according to the
+[postgres documentation](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING),
```

Comment:
I believe the mariadb and sql DBs use the same format of connection strings, but to make this documentation apply to the other engines, should we add a note about "Refer to the MariaDB or MySql docs for how their connection strings should be formatted"?